base_agent/sql_memory.py

```diff
@@ -416,8 +420,9 @@ def tag(self, subj_memid: str, tag_text: str):
             >>> tag_text = "shiny"
             >>> tag(subj_memid, tag_text)
         """
-        self.add_triple(subj=subj_memid, pred_text="has_tag", obj_text=tag_text)
+        return self.add_triple(subj=subj_memid, pred_text="has_tag", obj_text=tag_text)
 
+    # TODO remove_triple
     def untag(self, subj_memid: str, tag_text: str):
         """Delete tag for subject
 
```

Comment:
We should either remove `Returns` in docstring or return the number of rows affected.
(in a separate PR maybe)